### PR TITLE
Add Project Loved post

### PR DESCRIPTION
**Pull request by cl8n**
not complete but ready for review

missing:
- topic IDs
- other creators (currently only one creator is listed per map)

intro probably needs to be re-written too because it's just a re-hash of the first April one lol

### CODE REVIEW

File: news/2018-05-06-project-loved-week-of-may-6th.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2018-05-06-project-loved-week-of-may-6th/osu/fukagyaku-replace.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[MY FIRST STORY - Fukagyaku Replace](https://osu.ppy.sh/beatmapsets/227021) by [Moway](https://osu.ppy.sh/users/1352672)  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+With a staggering 791 favourites, [Moway](https://osu.ppy.sh/users/1352672)'s [Fukagyaku Replace](https://osu.ppy.sh/beatmapsets/227021) is insanely popular, and it’s easy to see why. The mapset is incredibly clean, featuring an easily rankable spread of difficulties by several notable mappers and a top difficulty which has been featured in too many tournaments to count. Hopefully this beloved map can finally receive the scoreboard it deserves through Loved!
```
Comment (MegaApplePi) [inline news/2018-05-06-project-loved-week-of-may-6th.md]:
```diff
- and it’s easy
+ and it's easy
```
weird

File: news/2018-05-06-project-loved-week-of-may-6th.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+[![](/wiki/shared/news/2018-05-06-project-loved-week-of-may-6th/mania/the-battle-of-lil-slugger-ch-1-boss-extended-cut-250bpm-.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Danny Baranowsky - The Battle of Lil' Slugger (Ch 1 Boss Extended Cut) (250bpm)](https://osu.ppy.sh/beatmapsets/79839) by [Staiain](https://osu.ppy.sh/users/86188)  
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+Up next we have a classic chart from the hands of the resident speed demon viking [Staiain](https://osu.ppy.sh/users/86188) and legendary American StepMania player [samurai7694](https://osu.ppy.sh/users/1033229), and it is a hard one.
+
+Starting out fairly tame, the chart slowly introduces what it's going to be about with short bursts of 250 bpm streams and then a longer 375 BPM stream which is the meat of the chart. It's fairly lenient patterning for singlestreams but clocking at whooping 375 BPM and runs of semi-dense jumpstream at 250 BPM for the most part. The speed factor is why this chart has been very popular both on SM and osu!mania for a very long time, and as such, I think it deserves its spot in the Loved category!
```
Comment (MegaApplePi) [inline news/2018-05-06-project-loved-week-of-may-6th.md]:
```diff
- 250 bpm streams
+ 250 BPM streams
```
just to be consistent?

File: news/2018-05-06-project-loved-week-of-may-6th.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+date: 2018-05-06 12:00:00 +0000
+---
+
+Another Sunday marks another [Project Loved](https://osu.ppy.sh/community/forums/120) voting on the horizon! Come and vote for your favourite beatmaps nominated from all four game modes!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello and welcome to the first [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of May 2018! This week contains several exciting picks for every game mode, and has something for both new and old players!
+
+For those of you who are unfamiliar with Project Loved or how the new system works, each mode has "captains" who pick a selection of your submissions to be put up for vote every two weeks. These chosen selections are each given their own community poll, and any beatmap with an approval rating above 75% after a week's time is moved into the Loved category. Beatmaps that are Loved have leaderboards, but do not award performance points (much like Qualified maps).
```
Comment (MegaApplePi) [inline news/2018-05-06-project-loved-week-of-may-6th.md]: `rating above 75%`
and 80% for osu!mania?

File: news/2018-05-06-project-loved-week-of-may-6th.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ Another Sunday marks another [Project Loved](https://osu.ppy.sh/community/forums
 
 Hello and welcome to the first [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of May 2018! This week contains several exciting picks for every game mode, and has something for both new and old players!
 
-For those of you who are unfamiliar with Project Loved or how the new system works, each mode has "captains" who pick a selection of your submissions to be put up for vote every two weeks. These chosen selections are each given their own community poll, and any beatmap with an approval rating above 75% after a week's time is moved into the Loved category. Beatmaps that are Loved have leaderboards, but do not award performance points (much like Qualified maps).
+For those of you who are unfamiliar with Project Loved or how the new system works, each mode has "captains" who pick a selection of your submissions to be put up for vote every two weeks. These chosen selections are each given their own community poll, and any beatmaps with an approval rating above 75% (or 80%, for osu!mania) after a week's time are moved into the Loved category. Beatmaps that are Loved have leaderboards, but do not award performance points (much like Qualified maps).
```
Comment (Toybickler) [inline news/2018-05-06-project-loved-week-of-may-6th.md]: osu!catch also is now 80% approval, should mention something about that change

**Pull request merged**